less/index.less

```diff
@@ -52,8 +52,209 @@ table {
 * {
     box-sizing: border-box;
 }
+/**** Variables ****/
+
+@defaultbg: #808080;
+
+/**** Variables ****/
+
+/***** Mixins ****/
+
+.center() {
+	display: flex;
+	justify-content: center;
+  }
+
+  .normalize(@decoration, @color) {
+	text-decoration: @decoration;
+	color: @color;
+  }
+
+/**** Mixins *****/
+
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 62.5%;
+	background-color: black;
+
+}
+
+.container {
+	max-width: 880px;
+	width: 100%;
+	margin: 0 auto;
+	padding: 0 2.5%;
+	background-color: white;
+}
+
+header {
+	width: 100%;
+	display: flex;
+}
+
+
+
+p {
+	font-size: 1.4rem;
+	line-height: 1.5;
+}
+
+a {
+	font-size: 1.4rem;
+	line-height: 1.5;
+	
+}
+
+h1 {
+	font-size: 4rem;
+	margin-top: 13px;
+	margin-bottom: 13px;
+}
+
+h2 {
+    font-size: 1.5rem;
+}
+
+h3 {
+	font-size: 3rem;
+}
+
+h4 {
+    font-size: 2rem;
+}
+
+li {
+	font-size: 1.4rem;
+}
+
+// button { ----- Removed this feature from site ------
+// 	background-color: @defaultbg;
+// 	border-radius: 6px;
+//     height: 37px;
+//     width: 14.9%;
+//     font-family: inherit;
+//     font-size: 15px;
+// 	margin-top: 15px;
+// 	color: white;
+// 	margin-bottom: 65px;
+// } ----- Removed this feature from site ------
+
+
+nav {
```

Comment:
Good job with the nesting here. You demonstrated you understand the concept of the material (and I know you said you feel this is much more intuitive and easier to read) as someone who reviews code all day... I agree! I'm happy you guys are using this now